WcaOnRails/config/locales/en.yml

```diff
@@ -1114,6 +1116,7 @@ en:
       contact: "Contact"
       information: "Information"
       events: "Events"
+      main_event: "Main event"
```

Comment:
Is this used? Travis seems to think it is not? https://travis-ci.org/thewca/worldcubeassociation.org/builds/576503852#L4951-L4963

Reply:
It's not used (yet,) I'll remove it for now.